shell/platform/tizen/tizen_view_elementary.cc

```diff
@@ -0,0 +1,354 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "tizen_view_elementary.h"
+
+#include <efl_extension.h>
+#include <ui/efl_util.h>
+
+#include "flutter/shell/platform/tizen/flutter_tizen_view.h"
+#include "flutter/shell/platform/tizen/logger.h"
+
+namespace {
+
+static const int kScrollDirectionVertical = 0;
+static const int kScrollDirectionHorizontal = 1;
+static const int kScrollOffsetMultiplier = 20;
+
+uint32_t EvasModifierToEcoreEventModifiers(const Evas_Modifier* evas_modifier) {
+  uint32_t modifiers = 0;
+  if (evas_key_modifier_is_set(evas_modifier, "Control")) {
+    modifiers |= ECORE_EVENT_MODIFIER_CTRL;
+  }
+  if (evas_key_modifier_is_set(evas_modifier, "Alt")) {
+    modifiers |= ECORE_EVENT_MODIFIER_ALT;
+  }
+  if (evas_key_modifier_is_set(evas_modifier, "Shift")) {
+    modifiers |= ECORE_EVENT_MODIFIER_SHIFT;
+  }
+  return modifiers;
+}
+
+void EvasObjectResize(Evas_Object* object, uint32_t width, uint32_t height) {
+  evas_object_resize(object, width, height);
+  evas_object_size_hint_min_set(object, width, height);
+  evas_object_size_hint_max_set(object, width, height);
+}
+
+}  // namespace
+
+namespace flutter {
+
+TizenViewElementary::TizenViewElementary(uint32_t width,
+                                         uint32_t height,
+                                         Evas_Object* parent)
+    : TizenView(width, height), parent_(parent) {
+  if (!CreateView()) {
+    FT_LOG(Error) << "Failed to create a platform view.";
+    return;
+  }
+
+  RegisterEventHandlers();
+  PrepareInputMethod();
+  Show();
+}
+
+TizenViewElementary::~TizenViewElementary() {
+  UnregisterEventHandlers();
+  DestroyView();
+}
+
+bool TizenViewElementary::CreateView() {
+  elm_config_accel_preference_set("hw:opengl");
+
+  int32_t parent_width, parent_height;
+  evas_object_geometry_get(parent_, nullptr, nullptr, &parent_width,
+                           &parent_height);
+
+  if (initial_width_ == 0) {
+    initial_width_ = parent_width;
+  }
+  if (initial_height_ == 0) {
+    initial_height_ = parent_height;
+  }
+
+  container_ = elm_table_add(parent_);
```

Comment:
Could you please leave comments somewhere why we need a container in the form of a table?

Reply:
The elm_table is one of the ways that can be used at C API level when overlapping widgets in elementary.
I don't think this is a special thing enough to reason leaving a comment.

Reply:
The container can eventually reach the user via the `FlutterDesktopViewGetEvasObject` under current implementations. 
Tables typically provide complex functionality for their unique layout flow. so it's generally considered to be ineffective when simply it occupying one box area, although not fatal. therefore, the reason for choosing a table among the various widgets should be reasonable.

Also, I think we have a duty to inform the user what the object is essentially returned through the `FlutterDesktopViewGetEvasObject`.

Reply:
I don't think that.
The API name and Description describe that XXXGetEvasObject returns Evas_Object.
I do not want to express that the returned object has the properties of Table, and I do not want the user to use the elm_table API.
The user should only be able to use the properties that Evas_Object can have.